react/features/app/getRouteToRender.web.ts

```diff
@@ -71,7 +76,7 @@ function _getWebConferenceRoute(state) {
  * @param {Object} state - The redux state.
  * @returns {Promise<Object>}
```

Comment:
The type is inferred by TypeScript correctly on this function. The `@returns` in the JSDoc comment is not used by TS

Reply:
Seems it's inffering correctly because the `route` objects from the calling functions are always initialised wih _getEmptyRoute() which defines the shape. Guess it's fine like this.